File: api/Entity/Workspace.cs
namespace api.Entity;

public class Workspace
{
    public int Id { get; set; }
    public int OwnerId { get; set; }
    public User Owner { get; set; }
    public string Name { get; set; }
    public string? Description { get; set; }
    public DateTime CreatedAt { get; set; }
    
    public List<WorkItem> Items { get; set; } = new List<WorkItem>();
}

File: client/src/type/Workspace.ts
export type Workspace  = {
  id: number
  name: string
  description: string
  createdAt: string
  owner: Owner
  items: ItemView[]
}

export type Owner = {
  id: number
  name: string
}

export type ItemView  = {
  id: number
  title: string
  status: number
}